### PR TITLE
Bug Fix #3255

### PR DESCRIPTION
**Pull request by taggie05**
Fixes #3255

 Include a change request file using $ npm run change
Description of changes

I changed overflow-y from auto to hidden.
I'm not sure if this is the correct file but the problem with the scroll bar is this issue. When you remove both overlfow-x and overflow-y the scroll bar disappears. If you change the overflow-y to hidden from auto it changes as well. I found the problem through the CSS. The link at the bottom is the CSS that I think the problem is connected to but I'm not 100% sure to be honest.

https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Callout/Callout.scss

### CODE REVIEW

**Comment by gokunymbus** [comment]
Perfect!

File: packages/office-ui-fabric-react/src/components/Callout/Callout.scss [hunk 1]
```diff
@@ -42,7 +42,7 @@ $Callout-smallbeak-slant-width: 16px;
 .main {
   background-color: $ms-color-white;
   overflow-x: hidden;
-  overflow-y: auto;
```
Comment (gokunymbus) [inline packages/office-ui-fabric-react/src/components/Callout/Callout.scss:45]: Would you mind providing a before and after screenshot? I just wan't to make sure this fixes the problem. Thank you!!

**Review by gokunymbus — CHANGES_REQUESTED** [review]

File: packages/office-ui-fabric-react/src/components/Callout/Callout.scss (hunk 1 above)
Comment (taggie05) [reply packages/office-ui-fabric-react/src/components/Callout/Callout.scss:45]: When I change the this is what I get on my local machine when I do the changes

**Review by gokunymbus — APPROVED** [review]

**Pull request closed without merging**